loadtest/control/utils.go

```diff
@@ -57,16 +57,28 @@ func RandomizeUserName(name string) string {
 }
 
 func emulateUserTyping(t string, cb func(term string) UserActionResponse) UserActionResponse {
+	r := rand.New(rand.NewSource(time.Now().Unix()))
+	typingSpeed := time.Duration(100+r.Intn(200)) * time.Millisecond // 100-300ms
+
 	runes := []rune(t)
 	var term string
 	var resp UserActionResponse
-	for i := range runes {
-		time.Sleep(time.Duration(rand.Intn(10)) * 1e1 * time.Millisecond)
+	for i := 0; i < len(runes); i++ {
+		time.Sleep(typingSpeed)
 		term += string(runes[i])
 		resp = cb(term)
 		if resp.Err != nil {
 			return resp
 		}
+		// %15 probability of mistyping, adds a rune than it will be overridden by
```

Comment:
0.15%

// 0.15% probability of mistyping. Add a rune which will be overridden
// by next iteration.

Reply:
Ah, yes it is. In Turkish you put "%" before the number :)